scripts/generate-cocoa-bindings.ps1

```diff
@@ -28,12 +28,49 @@ if (!(Get-Command sharpie -ErrorAction SilentlyContinue))
 # Ensure Xamarin is installed (or sharpie won't produce expected output).
 if (!(Test-Path '/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/64bits/iOS/Xamarin.iOS.dll'))
 {
-    Write-Output 'Xamarin.iOS not found. Attempting to install via Homebrew.'
-    brew install --cask xamarin-ios
+    Write-Output 'Xamarin.iOS not found. Attempting to install manually.'
+    
+    # Download Xamarin.iOS package
+    $packageName = 'xamarin.ios-16.4.0.23.pkg'
+    $directDownloadUrl = 'https://github.com/jamescrosswell/xamarin-ios/releases/download/16.4.0.23/Xamarin.iOS.16.4.0.23.pkg'
```

Comment:
@bruno-garcia I've created a temporary repo under my own GitHub account for the time being. We might want to move this into an official "Sentry" repo, incase I ever go postal. If so, let me know when we have somewhere more appropriate to host this file and we can update the download URL in this PR before merging.

> [!NOTE]
> At the moment it's stored as a [release artefact](https://github.com/jamescrosswell/xamarin-ios/releases/tag/16.4.0.23) in an otherwise empty repo